adage/analyze/management/commands/create_or_update_participation_type.py

```diff
@@ -0,0 +1,58 @@
+"""
+This management command will save new ParticipationType objects to the
```

Comment:
To be more accurate, this command actually saves **a** new Participation object (instead of **objects**), right?